siaskynet/skynet.py

```diff
@@ -44,9 +44,23 @@ def UploadFileRequest(path, opts=None):
             r = requests.post(url, files={opts.portalFileFieldname: f})
         return r
 
+    @staticmethod
+    def UploadFileRequestWithChunks(path, opts=None):
+        if opts is None:
+            opts = Skynet.default_upload_options()
+
+        charset = string.ascii_lowercase
+        uuid = ''.join(random.choice(charset) for _ in range(16))
+
+        r = requests.post("%s/%s/%s?filename=%s" % (opts.portalUrl, opts.portalUploadPath, uuid, opts.customFilename), data=path, headers={'Content-Type': 'application/octet-stream'})
```

Comment:
Could you drop the UUID please? We don't use that anymore as we now support auto UUID.
I don't think the other upload methods use the uuid.

Reply:
@Kcchouette if you could just make this change, I'll merge it. Looks good otherwise. Thanks for adding the missing `Close`s